README.md

```diff
@@ -401,7 +409,123 @@ via [additional environment variables](https://github.com/FIWARE/helm-charts/blo
+the OIDC4VCI-Protocol to compliant wallets.
+* [PDP](https://github.com/fiware/dsba-pdp) is an implementation of a Policy-Decision Point, evaluating Json-Web-Tokens containing VerifiableCredentials in a
+DSBA-compliant way. It also supports the evaluation in the context of i4Trust.
+* [Keyrock](https://github.com/ging/fiware-idm) is the FIWARE component responsible for Identity Management. Using Keyrock (in conjunction with other security
+components) enables you to add OAuth2-based authentication and authorization security to your services and applications.
+* [Kong Plugins](https://github.com/fiware/kong-plugins-fiware) allow to extend the API Gateway Kong by further functionalities required for FIWARE-based environments. Kong
+Gateway is a lightweight, fast, and flexible cloud-native API gateway. An API gateway is a reverse proxy that lets you
+manage, configure, and route requests to your APIs.
+
+#### How to deploy
+
+The recommended way of deployment is via the provided [Helm charts](https://github.com/DOME-Marketplace/iam-components).
+
+To deploy a setup,
+the [umbrella chart](https://helm.sh/docs/howto/charts_tips_and_tricks/#complex-charts-with-many-dependencies) of the
+iam-components can be used as followed:
+
+- create a configuration values file according to the own environment, as described [here](#how-to-configure-1).
+- add helm chart repository to helm installation
+  ```
+    helm repo add dome-iam https://dome-marketplace.github.io/iam-components
+    helm repo update
+  ```
+  > :bulb: All releases of the IAM-components reside in the helm-repository https://dome-marketplace.github.io/iam-components.
+  In addition to that, all Pre-Release versions(build from the Pull Requests) are provided in the
+  pre-repo https://dome-marketplace.github.io/iam-components/pre. The pre-repo will be cleaned-up from time to time, in
+  order to keep the index manageable.
+
+- install the components using the prepared configuration
+  ```
+    helm install <RELEASE_NAME> dome-iam/iam-components --namespace <NAME_SPACE> --version <CHART_VERSION> -f values.yaml
+  ```
+
+#### How to configure
+
+Before deploying these components, you need to update some values:
+
+* `rbac` and `serviceAccount`: Depending on your requirements, you might need to adapt settings for RBAC and service
+  account
+* `ingress` or `route`: You need to set up these settings to make a component externally accessible
+* `did`s of participants: Replace/add the DIDs of the issuer and other participants
+* Provide correct key in [keyfile.json](https://github.com/DOME-Marketplace/iam-components/blob/main/charts/iam-components/templates/keycloak.yaml)
+  for your issuer
+* `keycloak.frontendUrl`: Externally accessible address of the keycloak (should be the same as defined in ingress/route)
+* `keycloak.realm`: Adapt clients, users and roles according to your needs
+* `tir.com`: replace everywhere with actual TIR
```

Comment:
Might be confusing as this is a value and the others are field names. But on the other side the intro says "values"... not sure